tests/data/snake_oil_data.py

```diff
@@ -54,10 +66,10 @@ def content(self):
                         },
                         {
                             "children": [],
-                            "name": "default3",
+                            "name": ensemble_for_axis_labels_tests,
```

Comment:
Is it possible to just use the hard-coded strings for now? Or just use a different variable name, something like `ensemble_name_3`

Reply:
yeah sure, I can do `ensemble_name_3` : )

Reply:
changed the naming to be a bit less specific.